### PR TITLE
Fix: activePanel not updated if props changes

### PR DESCRIPTION
**Pull request by batFormat**
Не обновлялось состояние при изменениях пропса `activePanel` передаваемого в Page компонент.

### CODE REVIEW

File: src/Page.js [hunk 1]
```diff
@@ -198,6 +198,12 @@ class Page extends React.Component {
 		window.history.replaceState({ panel: homePanel }, homePanel);
 	}
 
+	UNSAFE_componentWillReceiveProps(nextProps) {
```
Comment (hit2hat) [inline src/Page.js]: This method is UNSAFE

File: src/Page.js [hunk 2]
```diff
@@ -198,6 +198,12 @@ class Page extends React.Component {
...
+		if (nextProps.activePanel !== this.state.activePanel) {
```
Comment (hit2hat) [inline src/Page.js]: Use deconstruction for state's objects.
`const { activePanel } = this.state;`

File: src/Page.js (hunk 1 above)
Comment (hit2hat) [reply src/Page.js]: Use [this](https://ru.reactjs.org/docs/react-component.html#static-getderivedstatefromprops) method

**Review by hit2hat — COMMENTED** [review]
Fix problems

**Comment by batFormat** [comment]
Sry. My bad. I'll fix it now.

**Pull request merged**